ci/cloudbuild/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG NODE_VERSION=14
+FROM node:${NODE_VERSION}-alpine
+
+RUN apk add --no-cache git bash curl tar python3
```

Comment:
Mostly out of curiosity, why do we need Python?

Reply:
For gcloud sdk